CMakeLists.txt

```diff
@@ -10,13 +10,9 @@ OPTION(PopSift_USE_POSITION_INDEPENDENT_CODE "Generate position independent code
 OPTION(PopSift_USE_GRID_FILTER "Switch off grid filtering to massively reduce compile time while debugging other things." ON)
 OPTION(PopSift_USE_NORMF "The __normf function computes Euclidian distance on large arrays. Fast but stability is uncertain." OFF)
 OPTION(PopSift_USE_TEST_CMD "Add testing step for functional verification" OFF)
-OPTION(PopSift_BOOST_USE_STATIC_LIBS "Link with static Boost libraries" OFF)
+OPTION(PopSift_BOOST_USE_STATIC_LIBS "Link examples with static Boost libraries" OFF)
```

Comment:
This can be moved inside the project for the examples, as it can be used as an independent project

Reply:
Moved to examples.